src/go/rdctl/pkg/factoryreset/empty.go

```diff
@@ -0,0 +1,18 @@
+//go:build darwin || linux
```

Comment:
Better to say anything-not-Windows:
```suggestion
//go:build !windows
```